projects/ngx-scrollbar/smooth-scroll/src/smooth-scroll-manager.ts

```diff
@@ -83,9 +84,17 @@ export class SmoothScrollManager {
    */
   private _initSmoothScroll(el: HTMLElement): Subject<void> {
     if (this._onGoingScrolls.has(el)) {
-      this._onGoingScrolls.get(el).next();
+      const elOngoingScrolls = this._onGoingScrolls.get(el);
```

Comment:
No need for the extra check, the element already checked here `this._onGoingScrolls.has(el)`

Reply:
I agree, but the compiler does not once strictNullChecks is enabled because the `.get()` can return null and I agree it's kind of stupid after having checked it explicitly :p